src/common/service/NetRemoteDataStreamingReactors.hxx

```diff
@@ -106,6 +107,7 @@ private:
     uint32_t m_numberOfDataBlocksToStream{};
     uint32_t m_numberOfDataBlocksWritten{};
     Microsoft::Net::Remote::DataStream::DataStreamOperationStatus m_writeStatus{};
+    std::atomic<bool> m_isCancelled{};
```

Comment:
nit: spelling (prefer US over British 😀):

```suggestion
    std::atomic<bool> m_isCanceled{};
```

Reply:
I didn't even realize that was a British spelling 😆 I will update the spelling throughout the PR

Reply:
Being Canadian 🍁, I normally use British spellings and my wife gives me a hard time about it, so I spot these pretty quickly 😆